tests/drone/drone.py

```diff
@@ -1,27 +1,27 @@
 #!/usr/bin/env python3
 import asyncio
-from datetime import datetime, timedelta
 import pathlib
 import random
 import sys
+from datetime import datetime, timedelta
 
 examples_dir = pathlib.Path(__file__).resolve().parents[2] / "examples"
 sys.path.append(str(examples_dir))
 
-from utils import (
+from utils import (  # noqa: E402
     TEXT_COLOR_CYAN,
     TEXT_COLOR_DEFAULT,
     TEXT_COLOR_RED,
     TEXT_COLOR_YELLOW,
     build_parser,
 )
 
-from yapapi import Golem, NoPaymentAccountError, Task, WorkContext
-from yapapi import __version__ as yapapi_version
-from yapapi import windows_event_loop_fix
-from yapapi.log import enable_default_logger
-from yapapi.payload import vm
-from yapapi.rest.activity import BatchTimeoutError
+from yapapi import windows_event_loop_fix  # noqa: E402
```

Comment:
we can probably remove `windows_event_loop_fix` once we drop support for Python < 3.8